controlplane/kubeadm/internal/workload_cluster_conditions_test.go

```diff
@@ -186,6 +203,9 @@ func TestUpdateEtcdConditions(t *testing.T) {
 					{Type: controlplanev1.KubeadmControlPlaneMachineEtcdMemberHealthyV1Beta2Condition, Status: metav1.ConditionFalse, Reason: controlplanev1.KubeadmControlPlaneMachineEtcdMemberNotHealthyV1Beta2Reason, Message: "Etcd reports errors: some errors"},
 				},
 			},
+			expectedEtcdMembersAgreeOnMemberList:      false, // without reading members, we can make assumptions.
```

Comment:
Not sure if these comments are correct. Aren't we at the point where we already read members?